### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by PSGitHubUser1**
You have changed the URL of API so many people are facing problems. So I am doing this PR to update the readme file.

Now its correct.

### CODE REVIEW

**Comment by DvvCz** [comment]
No, that is a different self-hosted instance of this.

File: readme.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Copy-paste this into your markdown content, and that is it. Simple!
 Change the `?username=` value to your GitHub username.
 
 ```md
-[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
+[![Anurag's GitHub stats](https://github-readme-stats-git-masterrstaa-rickstaa.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
```
Comment (MitkoVtori) [inline readme.md:105]: This is @rickstaa's (main collaborator) fork branch of this repository.

**Review by MitkoVtori — CHANGES_REQUESTED** [review]
The problem is now fixed so everyone can return to the basic link. Rickstaa's fork won't be able to withstand a large amount of users.  Also, since this is rick's fork of this repo, there's absolutely no reason to change the link in the main readme.md

**Comment by rickstaa** [comment]
> The problem is now fixed so everyone can return to the basic link. Rickstaa's fork won't be able to withstand a large amount of users. Also, since this is rick's fork of this repo, there's absolutely no reason to change the link in the main readme.md

@PSGitHubUser1 thanks for your pull request. @MitkoVtori's conclusion above is however correct my fork can only sustain a limited number of people at the same time due to GitHub's Graphql limits. 😅 Closing this for now.

**Pull request closed without merging**